src/web/mjs/connectors/MangaNexus.mjs

```diff
@@ -0,0 +1,55 @@
+import Connector from '../engine/Connector.mjs';
+import Manga from '../engine/Manga.mjs';
+
+export default class MangaNexus extends Connector {
+
+    constructor() {
+        super();
+        super.id = 'manganexus';
+        super.label = 'MangaNexus';
+        this.tags = ['manga', 'webtoon', 'portuguese'];
+        this.url = 'https://manganexus.net';
+        this.key = 'M_dDS-6KdyeTRJm-ZYL7E';
+    }
+
+    async _getMangaFromURI(uri) {
+        const url = new URL(`/_next/data/${this.key}/manga/${uri.split('/').pop()}.json`, this.url);
+        const request = new Request(url, this.requestOptions);
+        const id = uri.split('/').pop();
```

Comment:
Duplicated code, maybe first initialize `const id = uri.split('/').pop();` and then use it in `new URL(...${id}...)`

Reply:
fixed